packages/web/src/components/nav/desktop/NavAudio.tsx

```diff
@@ -27,6 +30,31 @@ const messages = {
   claimRewards: 'Claim Rewards'
 }
 
+const validRewardIds: Set<ChallengeRewardID> = new Set([
+  'track-upload',
+  'referrals',
+  'ref-v',
+  'mobile-install',
+  'connect-verified',
+  'listen-streak',
+  'profile-completion',
+  'referred',
+  'send-first-tip'
+])
+
+/** Pulls rewards from remoteconfig */
+const useRewardIds = (
+  hideConfig: Partial<Record<ChallengeRewardID, boolean>>
+) => {
+  const rewardsString = useRemoteVar(StringKeys.CHALLENGE_REWARD_IDS)
+  if (rewardsString === null) return []
+  const rewards = rewardsString.split(',') as ChallengeRewardID[]
+  const filteredRewards: ChallengeRewardID[] = rewards.filter(
```

Comment:
maybe `activeRewards`? and the hook could be `useActiveRewardIds`?

Reply:
Good points - I copied this over from pages/audio-rewards-page/ChallengeRewardsTile.tsx so didn't look too closely at it.